cron/src/test/java/com/foursquare/service/TrendingCategoriesServiceImplTest.java

```diff
@@ -0,0 +1,190 @@
+package com.foursquare.service;
```

Comment:
https://pastebin.com/en8g3Nrj As I said it won't work for list size 2. Try this test, it fails.